pkg/framework/api/nodeinfo_podinfo.go

```diff
@@ -311,7 +311,7 @@ func (m *PodInfoMaintainer) ReservedPodsNum() int {
 func (m *PodInfoMaintainer) GetPods() []*PodInfo {
 	pods := make([]*PodInfo, 0, m.bePodsMayBePreempted.Len()+m.gtPodsMayBePreempted.Len()+len(m.neverBePreempted))
 	rangeSplay := func(s splay.Splay) {
-		s.Range(func(so splay.StoredObj) {
+		s.RangeNoOrder(func(so splay.StoredObj) {
```

Comment:
Do not modify underlying basic implementation for specific plugins

Reply:
Ok, I have cancelled this commit.

The reason I modified this part before was that `InterPodAffinity `and `PodTopologySpread `plugins need to frequently obtain nodeInfo pods, and the original design was to obtain all pods through the **dfs** method, which had **poor performance** due to frequent recursion. 

Maybe in the future I can try a more elegant method to optimize the performance of GetPods through another PR.